speccpu2017/run_speccpu

```diff
@@ -28,6 +28,15 @@ arguments="$@"
 run_status="Ran"
 gen_args_back="$@"
 disks_found=0
+installed=0
+speccpu_run_dir=""
+speccpu_run="/speccpu_run"
+speccpu_iso_mnt="/speccpu"
+speccpu_iso_file=""
+uploads=""
+give_usage=0
+use_mfs=""
+run_loca_selected=0
```

Comment:
Suggestion s/run_loca_/run_loc_/ - loc is the more common shorthand for location.

Reply:
Done.